templates/cisco_asa_show_asp_table_vpn-context_detail.template

```diff
@@ -0,0 +1,37 @@
+Value VPN_CTX (\S+)
+Value PEER_IP (\d+.\d+.\d+.\d+)
+Value POINTER (\S+)
+Value STATE (\S+)
+Value FLAGS (\S+)
+Value SA (\S+)
+Value SPI (\S+)
+Value GROUP (\d+)
+Value PKTS (\d+)
+Value BAD_PKTS (\d+)
+Value BAD_SPI (\d+)
+Value SPOOF (\d+)
+Value BAD_CRYPTO (\d+)
+Value REKEY_PKT (\d+)
+Value REKEY_CALL (\d+)
+Value VPN_FILTER (\S+)
+
+Start
+  ^VPN\sCTX\s+=\s${VPN_CTX}
```

Comment:
```suggestion
  ^VPN\s+CTX -> Continue.Record
  ^VPN\s+CTX\s+=\s+${VPN_CTX}
```

Prefer not to restrict spacing to a single space unless required, as that is something that could easily be inconsistent even by accident of the developers.

I think it is probably safer to record on the CTX than on filter, so move capture to entry opening instead of what we currently know as the closing.